src/internationalization/translations/th.json

```diff
@@ -195,5 +195,17 @@
     "buttonNeutral": "ไว้ทีหลัง",
     "buttonNegative": "ยกเลิก",
     "buttonPositive": "ตกลง"
+  },
+  "changePhoneNumber":{
+    "blankPhoneNumberWarning": "กรุณากรอกหมายเลขโทรศัพท์",
+    "invalidPhoneNumberWarning": "กรุณากรอกหมายเลขโทรศัพท์ที่ถูกต้อง",
+    "changePhoneNumber":"เปลี่ยนหมายเลขโทรศัพท์",
+    "enterPhoneNumber":"กรอกหมายเลขโทรศัพท์ใหม่",
+    "enterPhoneNumberDescription":"เราจะส่งรหัสยืนยันตัวตนไปให้คุณทางข้อความ อาจมีค่าบริการเพิ่มเติม",
```

Comment:
I think the translation could be clearer. The user may misunderstood.

Reply:
consider removing "อาจมีค่าบริการเพิ่มเติม"